database/migration/migrateData.ts

```diff
@@ -1088,6 +1079,35 @@ const migrateData = async () => {
     `)
   }
 
+  // v0.9.0
+  if (databaseVersionLessThan('0.9.0')) {
+    console.log('Migrating to v0.9.0...')
+
+    await DB.changeSchema(`
+      CREATE TABLE IF NOT EXISTS user_org_policy_template (
+        user_org_policy TEXT,
+        template_id INTEGER 
+      );
+    `)
```

Comment:
Let's make this v1.0.0

I've already release an 0.9.0 build. And I've targeted v1.0.0 in [my PR](#1106)

(Plus we're due a v1 release once this is done 😆)

Reply:
Also, can you add a console.log that states the change, as per the other migrations.

Something like:
```
console.log(
      ' - Create user_org_policy_template table for easier filtering for row-level permissions'
    )
```